specification/cognitiveservices/data-plane/AnomalyDetector/stable/v1.1/MultivariateAnomalyDetector.json

```diff
@@ -0,0 +1,987 @@
+              ],
+              "x-ms-examples": {
+                  "Detect anomaly with multivariate Model": {
+                      "$ref": "./examples/DetectAnomaly.json"
+                  }
+              }
+          }
+      },
+      "/multivariate/models/{modelId}/last/detect": {
+          "post": {
+              "responses": {
+                  "200": {
+                      "description": "Detection has successfully completed.",
+                      "schema": {
+                          "$ref": "#/definitions/LastDetectionResult"
+                      }
+                  },
+                  "default": {
+                      "description": "Error response.",
+                      "schema": {
+                          "$ref": "#/definitions/ErrorResponse"
+                      },
+                      "headers": {
+                          "x-ms-error-code": {
+                              "description": "Error code",
+                              "type": "string"
+                          }
+                      }
+                  }
+              },
+              "summary": "Detect anomalies in the last a few points of the request body",
+              "description": "Synchronized API for anomaly detection.",
+              "operationId": "LastDetectAnomaly",
+              "parameters": [{
+                      "name": "modelId",
+                      "in": "path",
+                      "required": true,
+                      "type": "string",
+                      "description": "Model identifier",
+                      "format": "uuid"
+                  }, {
+                      "name": "body",
+                      "required": true,
+                      "in": "body",
+                      "schema": {
+                          "$ref": "#/definitions/LastDetectionRequest"
+                      },
+                      "description": "Request for last detection"
+                  }
+              ],
+              "x-ms-examples": {
+                  "Detect anomaly with multivariate Model": {
+                      "$ref": "./examples/LastDetectAnomaly.json"
+                  }
+              }
+          }
+      }
+  },
+  "definitions": {
+      "ModelInfo": {
+          "required": ["endTime", "sourceFilePath", "startTime", "storageAccountName", "storageEndpointSuffix"],
+          "properties": {
+              "storageAccountName": {
+                  "type": "string",
+                  "description": "Source link to the input variables. Each variable should be a csv file with two columns, `timestamp` and `value`. By default, the file name of the variable will be used as its variable name.",
+                  "example": "multiadsample"
+              },
+              "storageEndpointSuffix": {
+                  "type": "string",
+                  "description": "",
+                  "example": "blob.core.windows.net"
+              },
+              "sourceFilePath": {
```

Comment:
why split the path into multiple components?

Reply:
fixed.